streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorNode.java

```diff
@@ -220,11 +221,11 @@ public void process(final Record<KIn, VIn> record) {
                 internalProcessorContext.recordContext().sourceRawValue()
             );
 
-            final ProcessingExceptionHandler.ProcessingHandlerResponse response;
+            final ProcessingExceptionHandler.Response processingExceptionResponse;
             try {
-                response = Objects.requireNonNull(
-                    processingExceptionHandler.handle(errorHandlerContext, record, processingException),
-                    "Invalid ProductionExceptionHandler response."
+                processingExceptionResponse = Objects.requireNonNull(
+                    processingExceptionHandler.handleError(errorHandlerContext, record, processingException),
+                    "Invalid ProcessingExceptionResponse response."
```

Comment:
It's now called `Response`. 

```suggestion
                    "Invalid ProcessingExceptionHandler response."
```

Reply:
fixed